book.toml

```diff
@@ -6,7 +6,8 @@ src = "."
 title = "OpenTitan Documentation"
 
 [output.html]
-fold = { enable = true, level = 1 }
+site-url = "book/"
+fold = { enable = true}
```

Comment:
Can't these changes be added in the previous commit where book.toml was introduced?